package.json

```diff
@@ -2,7 +2,7 @@
     "name": "vscode-motoko",
     "displayName": "Motoko",
     "description": "Motoko language support",
-    "version": "0.17.0",
+    "version": "0.17.2",
```

Comment:
We can combine this into 0.17.1 if that would make more sense?

Reply:
yeah, no problem